src/containers/ProfilePage/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,118 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
```

Comment:
are your teammates also naming their files src/containers/ProfilePage with caps like this?

Reply:
yes
![image](https://user-images.githubusercontent.com/30551053/99442068-d8a4a980-2929-11eb-97e6-6062033d1a10.png)